.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+known_third_party =
```

Comment:
Is any thing missing after the equal?

Reply:
right, i copied that file out of another repo, because it wasnt there, and forgot to adjust the value. @teddius the old precommit-config had a hook that added missing thirdparty packages into this config. Is there a reason that it isnt in the new precommit-config?